android/app/src/main/java/updated/mysterium/vpn/model/manual/connect/PriceLevel.kt

```diff
@@ -0,0 +1,5 @@
+package updated.mysterium.vpn.model.manual.connect
+
+enum class PriceLevel {
+    FREE, LOW, MEDIUM, HIGH
```

Comment:
Enum values should be on separate lines.

Reply:
Fix